apps/frontend/app/(website)/registry/route.tsx

```diff
@@ -0,0 +1,379 @@
+      aliases: "{prettier}",
+      classification: "framework",
+      displayName: "Prettier",
+      createdAt: "2024-04-30T18:30:10.055Z",
+      updatedAt: "2024-04-30T18:30:10.055Z",
+    },
+    {
+      id: 31,
+      title: "nuxt",
+      aliases: "{nuxt}",
+      classification: "framework",
+      displayName: "Nuxt",
+      createdAt: "2024-07-19T00:18:15.450Z",
+      updatedAt: "2024-07-19T00:18:15.450Z",
+    },
+    {
+      id: 33,
+      title: "node.js",
+      aliases: "{node,nodejs,node.js}",
+      classification: "framework",
+      displayName: "Node.js",
+      createdAt: "2024-11-10T06:57:19.520Z",
+      updatedAt: "2024-11-10T06:57:19.520Z",
+    },
+  ],
+};
+
+export async function GET(req: NextRequest) {
+  const url = new URL(req.url);
+  const { searchParams } = url;
+
+  const qParts: string[] = [];
+
+  // ?framework=title -> tag:alias1 tag:alias2 ...
+  const frameworkTitle = searchParams.get("framework");
+  if (frameworkTitle) {
+    const aliases = getFrameworkAliasesFromTags(frameworkTitle);
+    if (aliases.length) {
+      qParts.push(...aliases.map((a) => `tag:${a}`));
+    } else {
+      // Fallback: treat the input as a tag itself
+      qParts.push(`tag:${frameworkTitle}`);
+    }
+  }
+
+  // ?category=cat -> tag:cat
+  const category = searchParams.get("category");
+  if (category) {
+    qParts.push(`tag:${category}`);
+  }
+
+  // ?author=name -> by:name
+  const author = searchParams.get("author");
+  if (author) {
+    qParts.push(`by:${author}`);
+  }
+
+  if (qParts.length) {
+    const q = encodeURIComponent(qParts.join(" "));
+    return NextResponse.redirect(`https://app.codemod.com/registry/?q=${q}`);
+  }
+
+  return NextResponse.redirect("https://app.codemod.com/registry");
+}
+
+/**
+ * Looks up a framework by its title (case-insensitive), then returns its alias list.
+ * - Matches TAGS.Tag where classification === "framework" and title equals input (case-insensitive).
+ * - Parses the non-JSON alias string like "{a,b,\"c d\"}" into an array.
+ */
+function getFrameworkAliasesFromTags(titleInput: string): string[] {
+  const norm = normalize(titleInput);
+  const frameworks = TAGS.Tag.filter(
+    (t) => normalize(t.classification) === "framework",
+  );
+
+  // Try exact match by title first
+  let framework = frameworks.find((t) => normalize(t.title) === norm);
+
+  // If not found, also try displayName equality
+  if (!framework) {
+    framework = frameworks.find((t) => normalize(t.displayName) === norm);
+  }
+
+  // If still not found, optionally match by alias equality
+  if (!framework) {
+    framework = frameworks.find((t) =>
+      parseAliases(t.aliases).map(normalize).includes(norm),
+    );
+  }
+
+  return framework ? parseAliases(framework.aliases) : [];
+}
+
+/**
+ * Parses strings like:
+ *  "{next.js,nextjs,next}"
+ *  "{\"best practices\"}"
+ * into: ["next.js","nextjs","next"] or ["best practices"]
+ */
+function parseAliases(raw: string): string[] {
```

Comment:
I'd recommend we simply update the base JSON object's aliases prop to be an array of strings instead of parsing it in runtime